### PR TITLE
Translations and new files for reports

### PR DESCRIPTION
**Pull request by pangolp**
- `.yml` file for bug reports
- `.yml` file for suggestions
- Move README file inside `.github`
- Translation and creation of the file in Spanish.

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
 ![logo](static/images/logos/home.png)
 
+[English](README.md) | [Spanish](README_ES.md)
```
Comment (Helias) [inline .github/README.md:3]:
```suggestion
[🇬🇧 English](README.md) | [ 🇪🇸 Spanish](README_ES.md)
```

File: .github/README_ES.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+![logo](static/images/logos/home.png)
+
+[Ingles](README.md) | [Español](README_ES.md)
```
Comment (Helias) [inline .github/README_ES.md:3]:
```suggestion
[🇬🇧 Ingles](README.md) | [🇪🇸 Español](README_ES.md)
```

**Review by Helias — APPROVED** [review]

**Comment by pangolp** [comment]
Wait for me to make the changes.

**Comment by pangolp** [comment]
Done. Thanks.

**Pull request merged**